tests/test_assets.py

```diff
@@ -172,6 +172,27 @@ def build_lookup_generic_cases(asset_finder_type):
 
 class AssetTestCase(TestCase):
 
+    asset_attrs = ['sid',
```

Comment:
It seems like we may want to add this list to assets.py, rather than here in the tests.